### DISCUSSION TITLE
decrement operation

### DISCUSSION THREAD

**Discussion by vdkdamian** [netreconlab/Parse-Swift#86] [Q&A] [answered]
I see there is an operation called `increment`.
I would assume there is also an operation called `decrement`?

I can't seem to find it. Is this something that is missing or just not implemented yet?

**Comment by cbaker6** [comment, marked as answer]
ParseSwift only supports what the server supports. If you would like to see additional operations supported on the server, I recommend you open up related issues there. Supported operations on the server are here: https://github.com/parse-community/parse-server/blob/65e5879e42cefb4b462c9383af3705d027131242/src/Controllers/DatabaseController.js#L274-L309 and https://github.com/parse-community/parse-server/blob/65e5879e42cefb4b462c9383af3705d027131242/src/Controllers/SchemaController.js#L1600-L1620

Decrement can be easily achieved by incrementing by a negative number:

```swift
let incrementOperation = savedScore.operation.increment("points", by: -1)
```

**Reply by vdkdamian** [reply]
> ```swift
> 
> let incrementOperation = savedScore.operation.increment("points", by: -1)
> 
> ```
Ah yes! Of course, thanks!
